src/Processor/Gotenberg.php

```diff
@@ -159,6 +159,10 @@ public function getPdfFromString(string $html, array $params = [], bool $returnF
             $chromium->extraHttpHeaders($params['extraHttpHeaders']);
         }
 
+        if (isset($params['metadata'])) {
+            $chromium->metadata($params['metadata']);
```

Comment:
Thank you for your PR!

For BC reasons, this change can't go in `1.2` as is, because this metadata was introduce in [gotenberg-php v2.2.0](https://github.com/gotenberg/gotenberg-php/releases/tag/v2.2.0)
and currently we support v1 see https://github.com/pimcore/web-to-print-bundle/blob/f4c14bcbb4a122f6d99c93de4639b477f9d815e4/composer.json#L23

So the suggestion is to rebase the branch to 1.x and flag this change as improvement and add a compatibility layer to avoid metadata getting called in gotenberg-php v1, as it would crash due to `Method not found` error. 
It should be using [formValue](https://github.com/gotenberg/gotenberg-php/commit/66afd63556c756e6fad5e8764ba3cec04a72998a#diff-a40bd4ab25b53d2fc6cf017d993b38d84a06232528609663bdabdd2576848705R180) on v1.

Reply:
Hi @kingjia90, thanks for pointing that out. We missed the fact that the `metadata` method is not available on v1.
I updated the code to support both versions.